### PR TITLE
Enable bundler-cache for GitHub Actions template

### PR DESCRIPTION
**Pull request by andyw8**
This seems like a useful default.

Details: https://github.com/actions/cache/blob/main/examples.md#ruby---bundler

### CODE REVIEW

File: bundler/lib/bundler/templates/newgem/github/workflows/main.yml.tt [hunk 1]
```diff
@@ -11,6 +11,7 @@ jobs:
       uses: ruby/setup-ruby@v1
       with:
         ruby-version: <%= RUBY_VERSION %>
+        bundler-cache: true
```
Comment (deivid-rodriguez) [inline bundler/lib/bundler/templates/newgem/github/workflows/main.yml.tt:14]: Yes! But then the `gem install` && `bundle install` lines below can be removed :)

Comment (andyw8) [reply bundler/lib/bundler/templates/newgem/github/workflows/main.yml.tt:14]: Oh yes, updated.

**Review by deivid-rodriguez — APPROVED** [review]
Thanks @andyw8!

**Comment by andyw8** [comment]
Thanks to @swanson for the idea.

**Pull request merged**